---
title: "Multifunctional Lipid Probes Background"
author: "Gaelen Guzman"
format: 
  html:
    embed-resources: false
standalone: true
crossref: 
  ref-hyperlink: true # (default is true)
---


**This repository is still under construction, please forgive us for incomplete sections as we build features!**

Functionalized lipid probes are at the intersection of several niche biotechnologies. This overview will serve as a brief summary of the capabilities, nuances, and caveats of their utility. Additional resources and published review articles will be referenced throughout and linked below for greater detail.

## Introduction: Why study lipid interactomes?

* Because at its core biology is just complex chemistry, all biological reactions are mediated through the physical interaction of two or more molecules

* Lipid::Lipid interactomes can provide insight into which of these physical contacts mediate these membrane properties. Lipids are most commonly enmeshed within membrane bilayers and lipid-lipid interactions confer membrane properties such as rigidity or detergent resistance. 

* In recent decades, many select lipid species have gained recognition for being crucial signaling molecules. For one example, the sphingolipid sphingosine-1-phosphate (S1P) is potently recognized by an array of S1P receptors which cause T cells to extravasate and chemotax toward sites of infection. 

* Understanding the proteins that a given lipid interacts with can reveal great insight into the wide-ranging cellular signaling effects that a lipid can have. 

## Why are functional modifications necessary in studying lipid interactions?

Synthetic lipid analogs have greatly enabled the exploration of lipid biology. However, it may not be obvious why such analogs are such valuable tools. 

While, genetic editing has enabled vast advances in the study of nucleic acids and proteins, there has been no obvious parallel for studying lipids. Using CRISPR or *de novo* gene synthesis, one can subtly alter a sequence to enable visualization and pulldown without greatly affecting the structure or function of the target gene. To name a few examples, one can: mutate the active site of a protein to change substrate specificity, swap in a different subcellular localization sequence to translocate the target protein to a different organelle, swap domains to alter protein::protein binding, fuse the target protein to any number of fluorescent domains to visualize a protein's localization, or add an affinity handle to enable the enrichment of the target protein and any other protein interactors [@schultzChemicalToolsLipid2023].

Lipids, however, are an entirely different story altogether. Foremost of these differences is that lipids are the product of enzymatic synthesis and thus their structure cannot be directly modified within a living biological system. For much of the history of lipid research, studies utilized whole gene knockout or knock-in paradigms to alter the global lipid composition of the cell. Through these methods, we know much about the localization and broad behavior of lipid classes -- but the biological roles of select lipid species are historically challenging to explore [REF]. 

Further, lipids are significantly smaller than even the smallest protein -- composed of dozens of atoms, as opposed to hundreds or thousands -- and, thus, their biophysical properties are much more sensitive to modification. It is nearly impossible to fuse a lipid with a fluorescent protein (such as GFP) or classical affinity handle (such as an HA tag) without drastically affecting the biological activity of the lipid [REF]. 

Finally, each of the tens of thousands of distinct lipid structures is a part of a rapidly shifting metabolic network: any given lipid can be cleaved, modified, or converted into other lipid species within short time frames. Some lipids, such as sphingosine have half-lives in the minutes [REF].

For these reasons, synthetic analogs to native lipids have been a great utility toward probing the biological function of select lipid species. The chief modifications described in this repository (highlighted in the figure below) are: 
1. A diazirine ring 
2. A terminal alkyne
3. A coumarin photocage

These modifications allow for the stabilization of protein::lipid interactions and the affinity enrichment of protein::lipid conjugates, all with tight spatiotemporal control. These modifications are described in greater detail below.

:::{.border}
```{r setup, echo=FALSE, include=FALSE}
library(plotly)
library(tidyr)

# Load your image
image_url <- "LipidProbe/Structures/SphingosineStructure.png"

# Define hover regions (rectangles) with coordinates based on the top of the image
hover_regions <- data.frame(
  x_start = c(0.0, 0.7),  # Left side of rectangles (fraction of width)
  x_end = c(0.4, 0.9),    # Right side of rectangles (fraction of width)
  y_top = c(0.2, 0.3),    # Top side of rectangles (fraction of height, from top of image)
  y_bottom = c(0.4, 0.4), # Bottom side of rectangles (fraction of height, from top of image)
  text = c("Region 1 Info", "Region 2 Info")
)

# Create grid lines at every 10% for both x and y
grid_lines <- list()
for (i in seq(0, 1, by = 0.1)) {
  # Add vertical grid line
  grid_lines <- c(grid_lines, list(
    list(
      type = "line",
      x0 = i, x1 = i,
      y0 = 0, y1 = 1,
      line = list(color = "gray", width = 1, dash = "dot")
    )
  ))
  
  # Add horizontal grid line
  grid_lines <- c(grid_lines, list(
    list(
      type = "line",
      x0 = 0, x1 = 1,
      y0 = i, y1 = i,
      line = list(color = "gray", width = 1, dash = "dot")
    )
  ))
}

# Create the plot
plot <- plot_ly()

# Add each rectangle as a separate trace
for (i in 1:nrow(hover_regions)) {
  region <- hover_regions[i, ]
  x_coords <- c(region$x_start, region$x_end, region$x_end, region$x_start, region$x_start)
  y_coords <- c(1 - region$y_top, 1 - region$y_top, 1 - region$y_bottom, 1 - region$y_bottom, 1 - region$y_top)
  
  plot <- plot %>%
    add_trace(
      x = x_coords,
      y = y_coords,
      type = "scatter",
      mode = "lines",
      line = list(
        color = "blue",     # Border color
        width = 2           # Border width
      ),
      fill = "none",         # No fill color inside the rectangle
      hoverinfo = "text",    # Show text on hover
      text = region$text
    )
}

# Add the image
plot <- plot %>%
  layout(
    images = list(
      list(
        source = image_url,
        xref = "paper",  # Normalized to percentage
        yref = "paper",  # Normalized to percentage
        x = 0,           # Start at 0% of the plot width
        y = 1,           # Start at 100% of the plot height
        sizex = 1,       # Cover 100% of the plot width
        sizey = 1,       # Cover 100% of the plot height
        xanchor = "left",
        yanchor = "top",
        layer = "below"
      )
    ),
    xaxis = list(visible = FALSE, range = c(0, 1)),  # Normalized from 0 to 1
    yaxis = list(visible = FALSE, range = c(0, 1)),
    shapes = grid_lines # Add the grid lines to the layout
  )


```

```{r, echo=FALSE, warnings=FALSE}
#| height: 500px
#| width: 100%
# Render the plot
plot

```
:::

### Terminal Alkyne

* Affinity handle
* Affix fluorophore post-hoc

### Diazirine: A photo-activated crosslinker {#sec-DiazirinePlacement}

* Crosslinking motif activated by 355nm UV light. 
* Irradiation of the ring produces a reactive carbene radical which covalently bonds with any nearby molecule. This radical can form linkages by inserting into carbo-hydrogen, heteroatom-hydrogen (SH, NH~2~, or OH) or double bonds[@haberkantFatFabulousBifunctional2014].
* This chemical group is the smallest available photoactivatable crosslinking motif [@haberkantFatFabulousBifunctional2014]

References to include: 
* [@schultzFlashClickMultifunctionalized2022a]
* [@schultzChemicalToolsLipid2023]
* [@farleyTrifunctionalSphinganineNew2024a]
* [@hoglingerTrifunctionalLipidProbes2017]

### Coumarin Photocage: A photo-activated protection moiety

* Many native lipids of interest are rapidly synthesized, catabolized, and interconverted throughout the vast lipid metabolic network. 
* The coumarin cage is a photolabile protection group. Upon stimulation with 400nm light, this group is released to expose the native structure of the lipid headgroup.
* The exposure of the native headgroup allows for recognition by protein binders. For example, Hoglinger et al. showed in 2015 that the release of caged sphingosine resulted in potent lysosomal calcium release -- a finding which greatly expanded the understanding of the intracellular sphingosine signaling cascade[@hoglingerIntracellularSphingosineReleases2015a]. In this and other studies, sub-cellular irradiation using a scanning laser allows for the select spatial control of uncaging. While not compatible with large-scale interaction studies, this method enables the assessment of biological activity via fluorescence microscopy.
* An additional boon of using coumarin as the caging moiety is its inherent fluorescence: one can simply use a fluorescence microscope to rapidly assess the intercalation of the functionalized lipid into the target cell.

## Photoactivatable & Clickable Probes / Bifunctional Lipid Probes

* The conceptual precursor to the so-called trifunctional probes, these typically include only the diazirine and the terminal alkyne moieties. 

References to include: 
* [@haberkantFatFabulousBifunctional2014] - Haberkant et al. 2014


## Trifunctional Lipid Probes

References to include:
* 

## Interactomics Experimental schema

:::{.border}
![](SiteFigures/BaselineProteomics_ExperimentalDesign.png){group="Overview_gallery"}
:::

### Cell treatment

### Protein collection

### Mass spectrometry analysis

#### Spectral counting

* Isolating peptide-like ions in MS1 scan, fragmenting in MS2 scan to record peptide sequence, consolidating the sum total of individual scans for each peptide mapped to each protein. This sum can be used as a proxy for the absolute abundance of the protein when comparing two samples prepared in identical manners and run on the same LC-MS/MS protocol.

Pros:
* Widely used, though not strictly a direct measurement of relative abundance. Highly accessible, as label-free quantification is much less costly and sample preparation is much more simple.

Cons:
* Co-elution of high- and low-abundance peptides can cause significant issues in the number of peptides can be isolated in MS1. 
* Samples are necessarily run at different times. Instrument calibration can drift, contaminants in the liquid chromatography can build, and errors can accrue in the preparation of each sample. Unless great care is taken, substantial differences between MS runs can result in the reporting of artificial differences between the samples.


#### Isobaric mass tagging

* Chemically adding a mass tag to each peptide in a sample, pooling with other samples, running as combined sample group in a single MS run. As before, peptide isolation in MS1, fragmentation and sequence identification in MS2. Additional identification of fragmented mass tags in MS2 reveals the relative abundances of the peptide in each contributing sample. 

Pros:
* Because many (16+ in the case of Tandem Mass Tags^TM^) samples can be pooled in each experimental group, the total number of MS runs can be reduced. Alternatively, the pooled experimental group can be fractionated (typically orthogonally to the reverse phase chromatography used in the LC injection method; e.g. cation exchange) so that high-abundance and low-abundance proteins can be spaced out. 
  * For example, instead of performing 16 individual low-depth MS runs using label-free methods, one may run 6 fractionated MS runs of the pooled experimental group, resulting in significantly greater depth of MS1 peptide coverage.

Cons:
* Cost
* Increased number of sample preparation steps, higher probability to introduce variation in sample handling.
* Requirement for normalizing total protein input: when comparing +UV to -UV in diazirine enrichment, one necessarily must artificially combine disproportionate sample quantities. [We might not want to mention this here, it's a niche issue...]
* Need for greater precision in MS scanning -- more expensive instruments and outsourcing of sample analysis.


\br

(Here's an example of what references would look like on the site: [@thomasTrifunctionalLipidDerivatives2025; @schultzFlashClickMultifunctionalized2022a; @schultzChemicalToolsLipid2023])
