<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.42">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="dcterms.date" content="2025-02-04">
<meta name="keywords" content="Lipid-protein interactions, Proteomics, Interactomics, Database, Multifunctionalized lipids">

<title>The Lipid Interactome:</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="GG_2025_applicationNote_files/libs/clipboard/clipboard.min.js"></script>
<script src="GG_2025_applicationNote_files/libs/quarto-html/quarto.js"></script>
<script src="GG_2025_applicationNote_files/libs/quarto-html/popper.min.js"></script>
<script src="GG_2025_applicationNote_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="GG_2025_applicationNote_files/libs/quarto-html/anchor.min.js"></script>
<link href="GG_2025_applicationNote_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="GG_2025_applicationNote_files/libs/quarto-html/quarto-syntax-highlighting-2f5df379a58b258e96c21c0638c20c03.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="GG_2025_applicationNote_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="GG_2025_applicationNote_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="GG_2025_applicationNote_files/libs/bootstrap/bootstrap-c0367b04c37547644fece4185067e4a7.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">


</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
<div class="quarto-alternate-formats"><h2>Other Formats</h2><ul><li><a href="GG_2025_applicationNote.docx"><i class="bi bi-file-word"></i>MS Word</a></li></ul></div></div>
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">The Lipid Interactome:</h1>
<p class="subtitle lead">An interactive and open access platform for exploring cellular lipid-protein interactions</p>
</div>


<div class="quarto-title-meta-author">
  <div class="quarto-title-meta-heading">Authors</div>
  <div class="quarto-title-meta-heading">Affiliations</div>
  
    <div class="quarto-title-meta-contents">
    <p class="author">Gaelen Guzman <a href="mailto:gaelen.guzman@lipidinteractome.org" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> <a href="https://orcid.org/0000-0002-7696-6034" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Oregon Health &amp; Science University
          </p>
      </div>
    <div class="quarto-title-meta-contents">
    <p class="author">André Nadler <a href="https://orcid.org/0000-0001-6329-3910" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Max Plank Institute
          </p>
      </div>
    <div class="quarto-title-meta-contents">
    <p class="author">Frank Stein <a href="https://orcid.org/0000-0001-9695-1692" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            European Molecular Biology Laboratory
          </p>
      </div>
    <div class="quarto-title-meta-contents">
    <p class="author">Jeremy M. Baskin <a href="https://orcid.org/0000-0003-2939-3138" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Cornell University
          </p>
      </div>
    <div class="quarto-title-meta-contents">
    <p class="author">Carsten Schultz <a href="https://orcid.org/0000-0002-5824-2171" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Oregon Health &amp; Science University
          </p>
      </div>
    <div class="quarto-title-meta-contents">
    <p class="author">Fikadu Tafesse <a href="https://orcid.org/0000-0002-8575-4164" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Oregon Health &amp; Science University
          </p>
      </div>
  </div>

<div class="quarto-title-meta">

      
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">February 4, 2025</p>
    </div>
  </div>
  
    
  </div>
  

<div>
  <div class="keywords">
    <div class="block-title">Keywords</div>
    <p>Lipid-protein interactions, Proteomics, Interactomics, Database, Multifunctionalized lipids</p>
  </div>
</div>

</header>


<section id="summary" class="level2">
<h2 class="anchored" data-anchor-id="summary">Summary</h2>
<section id="abstract" class="level3">
<h3 class="anchored" data-anchor-id="abstract">Abstract</h3>
<p>Lipid-protein interactions play essential roles in cellular signaling and membrane dynamics, yet their systematic characterization has long been hindered by the inherent biochemical properties of lipids. Recent advances in functionalized lipid probes – equipped with photoactivatable crosslinkers, affinity handles, and photocleavable protecting groups – have enabled proteomics-based identification of lipid interacting proteins with unprecedented specificity and resolution. Despite the growing number of published lipid interactomes, there remains no centralized effort to harmonize, compare, or integrate these datasets.</p>
<p>The Lipid Interactome addresses this gap by providing a structured, interactive web portal that adheres to FAIR data principles – ensuring that lipid interactome studies are Findable, Accessible, Interoperable, and Reusable. Through standardized data formatting, interactive visualizations, and direct cross-study comparisons, this resource enables researchers to systematically explore the protein-binding partners of diverse bioactive lipids. By consolidating and curating lipid interactome proteomics data from multiple studies, the Lipid Interactome database serves as a critical tool for deciphering the biological functions of lipids in cellular systems.</p>
</section>
<section id="availability" class="level3">
<h3 class="anchored" data-anchor-id="availability">Availability</h3>
<p>This site can be viewed at <a href="lipidinteractome.org">LipidInteractome.org</a> all data is available for download. No user information is collected or necessary for data navigation, interaction, or download.</p>
</section>
<section id="data-submission-contact" class="level3">
<h3 class="anchored" data-anchor-id="data-submission-contact">Data Submission &amp; Contact</h3>
<p>If you would like to submit a relevant dataset, see our <a href="https://lipidinteractome.org/contactus/datasubmission">Data Submission Page</a> for details, contact info, and instructions. For site feedback and additional questions, please email the site administrators at <a href="mailto:Contact.Us@lipidinteractome.org" class="email">Contact.Us@lipidinteractome.org</a></p>
</section>
<section id="keywords" class="level3">
<h3 class="anchored" data-anchor-id="keywords">Keywords</h3>
<p>Lipid-protein interactions, proteomics, interactomics, database, multifunctionalized lipids, photoaffinity labeling</p>
</section>
</section>
<section id="introduction" class="level2">
<h2 class="anchored" data-anchor-id="introduction">Introduction</h2>
<p>Historically, lipids have been considered merely the passive structural components of cellular membranes. However, recent decades have revealed that this diverse category of biomolecule plays essential and active roles in the cell, ranging from modulating membrane dynamics, determining organelle identity, serving as catabolites for energy storage and metabolism, and bioactive signaling mediators for organellar and intracellular signaling<span class="citation" data-cites="barischMembraneDamageRepair2023 vanmeerMembraneLipidsWhere2008a holthuisLipidLandscapesPipelines2014a niphakisGlobalMapLipidBinding2015a chernomordikProteinLipidInterplayFusion2003 simonsLipidRaftsSignal2000a">[@barischMembraneDamageRepair2023; @vanmeerMembraneLipidsWhere2008a; @holthuisLipidLandscapesPipelines2014a; @niphakisGlobalMapLipidBinding2015a; @chernomordikProteinLipidInterplayFusion2003; @simonsLipidRaftsSignal2000a]</span>. Interactions with proteins are essential to many of these crucial lipid functions – and yet, lipid–protein interactions are extremely challenging to study: The small size and biophysical properties of lipids make conventional proteomics approaches unsuitable for mapping lipid interactors<span class="citation" data-cites="schultzChemicalToolsLipid2023">[@schultzChemicalToolsLipid2023]</span>. Unlike proteins, which can be readily tagged or immobilized for affinity-based interaction studies, lipids require specialized chemical modifications to enable selective and controlled interaction profiling<span class="citation" data-cites="schultzFlashClickMultifunctionalized2022a">[@schultzFlashClickMultifunctionalized2022a]</span>.</p>
<p>Recent advances in synthetic, multifunctionalized lipid analogs have revolutionized the field, enabling the systematic interrogation of lipid interactomes<span class="citation" data-cites="schultzFlashClickMultifunctionalized2022a">[@schultzFlashClickMultifunctionalized2022a]</span>. These probes incorporate strategically designed functional groups that enable covalent crosslinking, affinity enrichment, and controlled activation. Most commonly, these include: (1) a diazirine group for photoactivatable crosslinking upon exposure to 355 nm ultraviolet (UV) light, stabilizing non-covalent and often modest-affinity lipid–protein interactions; (2) a terminal alkyne, permitting click chemistry-based enrichment; and, in some cases, (3) a coumarin photocage, which shields the lipid probe from premature enzymatic modification until it is selectively cleaved by 405 nm light (a wavelength that does not induce diazirine photolysis)<span class="citation" data-cites="schultzFlashClickMultifunctionalized2022a schultzChemicalToolsLipid2023 hoglingerTrifunctionalLipidProbes2017 haberkantFatFabulousBifunctional2014">[@schultzFlashClickMultifunctionalized2022a; @schultzChemicalToolsLipid2023; @hoglingerTrifunctionalLipidProbes2017; @haberkantFatFabulousBifunctional2014]</span>. Together, these modifications enable direct proteomic identification of lipid interactors, distinguishing true binding partners from transient or non-specific associations<span class="citation" data-cites="haberkantFatFabulousBifunctional2014 schultzChemicalToolsLipid2023 farleyTrifunctionalSphinganineNew2024a">[@haberkantFatFabulousBifunctional2014; @schultzChemicalToolsLipid2023; @farleyTrifunctionalSphinganineNew2024a]</span>.</p>
<p>Numerous studies have already employed these lipid probes to elucidate the interactomes of key bioactive lipids, including diacylglycerol, sphingosine, sphinganine, phosphatidic acid, and phosphatidylethanolamine, N-acylphosphatidylethanolamine, and phosphatidyl alcohol<span class="citation" data-cites="hoglingerTrifunctionalLipidProbes2017 farleyTrifunctionalSphinganineNew2024a farleyTrifunctionalFattyAcid2024 thomasTrifunctionalLipidDerivatives2025 chiuPhotoaffinityLabelingReveals2025 yuChemoproteomicsApproachProfile2022 mullerSynthesisCellularLabeling2021">[@hoglingerTrifunctionalLipidProbes2017; @farleyTrifunctionalSphinganineNew2024a; @farleyTrifunctionalFattyAcid2024; @thomasTrifunctionalLipidDerivatives2025; @chiuPhotoaffinityLabelingReveals2025; @yuChemoproteomicsApproachProfile2022; @mullerSynthesisCellularLabeling2021]</span>. These initial studies have followed a comparative proteomics workflow, wherein crosslinked (“+UV”) and non-crosslinked (“-UV”) samples are analyzed side by side to minimize artifacts and confirm specificity<span class="citation" data-cites="haberkantFatFabulousBifunctional2014 schultzChemicalToolsLipid2023">[@haberkantFatFabulousBifunctional2014; @schultzChemicalToolsLipid2023]</span>. However, despite the growing number of lipid interactome datasets, no centralized resource has existed to compile, compare, and standardize these findings.</p>
<p>To address this gap, we present the <a href="https://www.lipidinteractome.org">Lipid Interactome</a> – a curated repository designed to facilitate the comparative analysis of lipid–protein interactions from multiple studies. This resource consolidates proteomics data from multifunctional lipid probes, enabling researchers to efficiently explore and contextualize lipid-binding proteins across different lipid species and cellular models. Currently, the repository hosts interactomes derived from trifunctionalized phosphatidic acid, phosphatidylethanolamine, phosphatidylinositol <em>tris</em>phosphate, N-acylphosphatidylethanolamine, phosphatidyl alcohol, sphingosine, and sphinganine in HeLa and Huh7 cells<span class="citation" data-cites="thomasTrifunctionalLipidDerivatives2025 mullerSynthesisCellularLabeling2021 chiuPhotoaffinityLabelingReveals2025 yuChemoproteomicsApproachProfile2022 farleyTrifunctionalSphinganineNew2024a">[@thomasTrifunctionalLipidDerivatives2025; @mullerSynthesisCellularLabeling2021; @chiuPhotoaffinityLabelingReveals2025; @yuChemoproteomicsApproachProfile2022; @farleyTrifunctionalSphinganineNew2024a]</span>. Additionally, it includes control datasets from two fatty acid probes to distinguish specific from non-specific lipid-binding proteins<span class="citation" data-cites="farleyTrifunctionalFattyAcid2024">[@farleyTrifunctionalFattyAcid2024]</span>. We additionally intend to expand this resource to include more complex experimental designs as they are published (e.g.&nbsp;Disease versus Healthy Control or Infection versus Mock).</p>
<p>By providing a centralized platform for lipid interactome data, the Lipid Interactome significantly lowers the barriers to entry for researchers investigating lipid signaling pathways, enabling hypothesis generation through comparative analysis of multiple studies. Overall, this repository represents an important step toward systematizing lipid–protein interaction studies, ensuring that bioactive lipids are afforded the same level of proteomic scrutiny as their protein counterparts.</p>
</section>
<section id="description" class="level2">
<h2 class="anchored" data-anchor-id="description">Description</h2>
<section id="overview" class="level3">
<h3 class="anchored" data-anchor-id="overview">Overview</h3>
<p>The&nbsp;Lipid Interactome&nbsp;is an interactive discovery platform built using&nbsp;Quarto, designed to centralize and enhance the study of lipid–protein interactions through curated datasets and dynamic visualization tools. Each page integrates&nbsp;R-generated figures, embedding scripts directly into the source material to ensure reproducibility and transparency. The repository currently hosts data from&nbsp;seven published studies, with an expandable framework that supports the incorporation of future datasets as the field progresses.</p>
<p>At its core, the&nbsp;Lipid Interactome&nbsp;enables the&nbsp;direct, qualitative comparison&nbsp;of lipid interactomes between studies, providing researchers with invaluable insights into the diverse roles that bioactive lipids play in cellular processes. By leveraging standardized data formatting and interactive visualization – including study-specific analysis pages, lipid probe aggregation, and cross-study comparisons via a&nbsp;Shiny&nbsp;application – the platform facilitates intuitive exploration of protein–lipid interactions. As a&nbsp;living repository, we actively encourage researchers to contribute their findings through our&nbsp;<a href="https://lipidinteractome.org/contactus/datasubmission">Data&nbsp;Submission</a>&nbsp;page, ensuring that this resource continues to evolve alongside advancements in lipidomics research.</p>
</section>
<section id="fair-data-principles" class="level3">
<h3 class="anchored" data-anchor-id="fair-data-principles">FAIR data principles</h3>
<p>The&nbsp;Lipid Interactome&nbsp;database was developed with the FAIR (Findable, Accessible, Interoperable, and Reusable) data principles in mind, ensuring that lipid–protein interaction data can be systematically explored, compared, and reused by the research community. Given the historical inconsistency in proteomics data curation, we have taken deliberate steps to standardize, annotate, and document our datasets for maximum utility.</p>
<section id="findable" class="level4">
<h4 class="anchored" data-anchor-id="findable">Findable</h4>
<p>To address the fragmented nature of lipid interactome studies, we have established a centralized platform where researchers can interact with the results of individual studies or explore aggregated findings across multiple studies on a given lipid. The repository is structured to facilitate intuitive navigation between experimental datasets, offering clear entry points for study-specific and lipid-specific analyses. Because this resource is intended as a living repository, we actively invite researchers to contribute their findings following the submission guidelines detailed on our&nbsp;<a href="https://lipidinteractome.org/contactus/datasubmission">Data Submission</a>&nbsp;page.</p>
</section>
<section id="accessible" class="level4">
<h4 class="anchored" data-anchor-id="accessible">Accessible</h4>
<p>All datasets within&nbsp;the Lipid Interactome&nbsp;are available for download in standardized&nbsp;<code>.csv</code>&nbsp;format, ensuring broad accessibility across different computational environments. To further enhance usability, we provide a&nbsp;<a href="https://www.lipidinteractome.org/background/multifunctionallipidprobesoverview">detailed&nbsp;overview&nbsp;of data&nbsp;generation&nbsp;and&nbsp;analysis</a>, including explicit descriptions of dataset structure, column definitions, and the inherent limitations of multifunctional lipid probes. Only peer-reviewed proteomics datasets are included in this repository to maintain high data quality and reliability.</p>
</section>
<section id="interoperable" class="level4">
<h4 class="anchored" data-anchor-id="interoperable">Interoperable</h4>
<p>Given the diversity of methodologies employed across lipid interactome studies, we have taken care to harmonize data formats and metadata annotations. While many studies share common analytical pipelines, others employ distinct mass spectrometry workflows. To ensure consistency, we have structured the data to allow parallel comparisons and provided&nbsp;<a href="https://www.lipidinteractome.org/background/proteomicsusingmultifunctionalprobes">documentation&nbsp;on&nbsp;methodological&nbsp;differences</a>. This standardization effort facilitates direct cross-study comparisons without requiring extensive reformatting by the end user. Each dataset is accompanied by metadata which describe the sample handling and data analysis techniques – with clarifying text describing how “significance” is determined for each analysis.</p>
</section>
<section id="reusable" class="level4">
<h4 class="anchored" data-anchor-id="reusable">Reusable</h4>
<p>To maximize the utility of these datasets, we provide extensive documentation detailing data acquisition, analysis workflows, and potential limitations. Wherever possible, technical jargon has been minimized to ensure accessibility to a broad audience, from lipid biologists to computational scientists. Additionally, we reference key publications that delve deeper into the synthesis of lipid probes and the specifics of each proteomics workflow, allowing researchers to further contextualize and validate their findings.</p>
<p>By adhering to these FAIR principles,&nbsp;the Lipid Interactome&nbsp;database aims to serve as a robust and enduring resource for the lipid biology community, reducing redundancy, improving data transparency, and accelerating the study of lipid-mediated signaling.</p>
</section>
</section>
<section id="data-visualization-and-download-modules" class="level3">
<h3 class="anchored" data-anchor-id="data-visualization-and-download-modules">Data Visualization and Download Modules</h3>
<p>To facilitate efficient exploration and analysis of lipid–protein interaction data, the&nbsp;Lipid Interactome&nbsp;repository incorporates a suite of interactive visualization and data access tools. These modules are designed to enhance user engagement, provide clear interpretability of experimental findings, and streamline data retrieval for further computational analysis – overall, the site is designed to ensure that the&nbsp;Lipid Interactome&nbsp;is not merely a static repository, but rather a dynamic and evolving resource for lipid biologists and bioinformaticians alike.</p>
<section id="study-specific-data-exploration" class="level4">
<h4 class="anchored" data-anchor-id="study-specific-data-exploration">Study-Specific Data Exploration</h4>
<p>Each publication contributing to the repository is provided with a dedicated results page, allowing users to examine study findings in detail. These pages include interactive&nbsp;Volcano,&nbsp;Ranked-Order, and&nbsp;MA plots, which enable visualization of differential protein enrichment across experimental conditions. The plots are rendered using the&nbsp;Plotly&nbsp;and&nbsp;htmlwidgetsR packages, ensuring a responsive, client-side experience that supports zooming, hovering, image export, and data filtering<span class="citation" data-cites="plotly ramnathvaidyanathanHtmlwidgetsHTMLWidgets2015">[@plotly; @ramnathvaidyanathanHtmlwidgetsHTMLWidgets2015]</span>. For transparency and reproducibility, all datasets associated with a given study are available for direct download.</p>
</section>
<section id="lipid-probe-specific-pages" class="level4">
<h4 class="anchored" data-anchor-id="lipid-probe-specific-pages">Lipid Probe-Specific Pages</h4>
<p>In addition to the pages for each independent publication, datasets are further organized by lipid probe, providing a consolidated view of protein interactors detected across multiple experiments and independent studies. Each lipid-specific page includes study metadata and cell type – facilitating cross-study comparisons. Future iterations of the repository will extend this functionality by enabling direct comparisons of interactomes obtained from different cell lines treated with the same lipid probe, further refining the contextual understanding of lipid–protein interactions.</p>
</section>
<section id="comparative-analysis-via-shiny-application" class="level4">
<h4 class="anchored" data-anchor-id="comparative-analysis-via-shiny-application">Comparative Analysis via Shiny Application</h4>
<p>Where study design permits, data has been integrated into an interactive&nbsp;Shiny&nbsp;application, allowing users to perform direct (though qualitative)&nbsp;probe-to-probe comparisons&nbsp;(<a href="https://www.lipidinteractome.org/probevsprobecomparison/enrichedhitscomparison">Probe&nbsp;vs&nbsp;Probe&nbsp;Comparisons</a>). This tool enables researchers to explore the similarity of lipid interactomes across experimental conditions via&nbsp;linear regression analysis, highlighting proteins that are significantly enriched in one dataset, both datasets, or neither. By providing an intuitive framework for interactome comparison, this feature aids in the identification of promising targets for further investigation.</p>
<p>Because each publication utilizes distinct sample handling and data analysis techniques, we emphasize that these comparisons are qualitative – we hope to discourage quantitative statements regarding relative enrichment between lipid probes (e.g.&nbsp;“Protein X is enriched 3x more to Lipid A than to Lipid B”). However, great utility lies in assessing co-enrichment (or orthogonal enrichment) of proteins to disparate lipid probes without quantitative comparison.</p>
</section>
</section>
<section id="data-format" class="level3">
<h3 class="anchored" data-anchor-id="data-format">Data format</h3>
<p>While there are numerous ways to quantify the mass spectrometry (MS) data, the majority of studies included in our repository have utilized the same (or highly similar) quantitative MS pipeline using data-dependent acquisition techniques, such as Tandem Mass Tagging<span class="citation" data-cites="thompsonTandemMassTags2003">[@thompsonTandemMassTags2003]</span>. These datasets include those reported by Farley et al.&nbsp;(2024), Farley et al.&nbsp;(2024), and Thomas et al.&nbsp;(2025)<span class="citation" data-cites="farleyTrifunctionalSphinganineNew2024a farleyTrifunctionalFattyAcid2024 thomasTrifunctionalLipidDerivatives2025">[@farleyTrifunctionalSphinganineNew2024a; @farleyTrifunctionalFattyAcid2024; @thomasTrifunctionalLipidDerivatives2025]</span>. Some studies have also featured quantitative MS analysis using stable isotope labeling of amino acids in cell culture (SILAC), which also enables extraction of key data described below<span class="citation" data-cites="chiuPhotoaffinityLabelingReveals2025 yuChemoproteomicsApproachProfile2022">[@chiuPhotoaffinityLabelingReveals2025; @yuChemoproteomicsApproachProfile2022]</span>. Correspondingly, the ideal data structure for incorporation into our repository will contain the information shown in <a href="#tbl-dataStructure" class="quarto-xref">Table&nbsp;1</a>.</p>
<div class="border">
<div id="tbl-dataStructure" class="striped quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-dataStructure-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;1: <strong>Example data table for smooth incorporation into Lipid Interactome.</strong> <strong><em>Column descriptions:</em></strong> <em>Gene name</em>: The identifier for genes identified in the dataset (or Accession Number). <em>Uniprot accession</em>: Stable identifiers of UniProtKB entries; integrates species and isoform data on the identified protein. <em>Enrichment value</em>: Fold change of Experimental versus Control fold-change, typically log2-transformed – ideally the product of TMT or other quantitative mass spectrometry methods. <em>p value</em>: the results of an ANOVA or LIMMA analysis, corrected for multiple hypothesis testing. <em>AveExpr</em>: An averaged value of the m/z ion intensity of the Experimental condition and Control condition used for the enrichment value (used to prepare MA plots for quality control). <em>FDR</em>: False-discovery rate of protein identification.
</figcaption>
<div aria-describedby="tbl-dataStructure-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<table class="table-striped caption-top table">
<thead>
<tr class="header">
<th style="text-align: left;">Gene name</th>
<th style="text-align: center;"><a href="https://www.uniprot.org/help/accession_numbers">Uniprot accession</a></th>
<th style="text-align: center;">logFC</th>
<th style="text-align: center;">p value</th>
<th style="text-align: center;">AveExpr</th>
<th style="text-align: center;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">GeneA</td>
<td style="text-align: center;">P80723</td>
<td style="text-align: center;">1.2</td>
<td style="text-align: center;">0.001</td>
<td style="text-align: center;">1.0x10<sup>18</sup></td>
<td style="text-align: center;">0.0001</td>
</tr>
<tr class="even">
<td style="text-align: left;">GeneB</td>
<td style="text-align: center;">P32004</td>
<td style="text-align: center;">0.1</td>
<td style="text-align: center;">0.6</td>
<td style="text-align: center;">1.0x10<sup>5</sup></td>
<td style="text-align: center;">0.005</td>
</tr>
<tr class="odd">
<td style="text-align: left;">…</td>
<td style="text-align: center;">…</td>
<td style="text-align: center;">…</td>
<td style="text-align: center;">…</td>
<td style="text-align: center;">…</td>
<td style="text-align: center;">…</td>
</tr>
</tbody>
</table>
</div>
</figure>
</div>
</div>
<p><br></p>
<p><strong>We wish to emphasize that we will collaborate with data providers to prepare visualizations which best suit their datasets – the example dataset above merely depicts an ideal case which allows for easy incorporation and cross-study comparison. As proteomics techniques have improved rapidly in recent years, we have done our best to incorporate all data structures into this repository.</strong></p>
<p>Additionally, we are working on providing visualization functionality for studies which utilized Peptide Spectral Matching (PSM) to report enrichment toward lipid probes. Similarly, data-independent acquisitio (DIA) is a quickly emerging alternative to data-dependent acquisition (DDA; e.g.&nbsp;TMT) <span class="citation" data-cites="birhanuMassSpectrometrybasedProteomics2023 boysClinicalApplicationsMass2023">[@birhanuMassSpectrometrybasedProteomics2023; @boysClinicalApplicationsMass2023]</span>; as with PSM datasets, we are working to incorporate DIA data formats.</p>
<p>At present, we are accepting datasets as analyzed by the original researchers, including protein identifications, enrichment quantification, and the results of statistical testing. Summaries of both source and data wrangling can be found in the <em>Study Information</em> dropdown section of each data page. Further, the raw data (as provided directly by authors or pulled from publicly-available datasts) and data wrangling scripts are available for download in the same dropdown tabs.</p>
</section>
</section>
<section id="discussion" class="level2">
<h2 class="anchored" data-anchor-id="discussion">Discussion</h2>
<p>The advent of functionalized lipid probes has fundamentally transformed lipid biology by enabling the proteomic identification of lipid-binding proteins with unprecedented specificity. However, as the number of published lipid interactome studies grows, the lack of a centralized framework for comparing and integrating these datasets presents a significant challenge for deciphering biological functions of individual lipids and lipid-binding proteins holistically and identifying trends across datasets. The Lipid Interactome was developed to address this need by providing researchers with a structured platform to explore, cross-reference, and build upon previous findings. By aggregating and standardizing lipid–protein interaction data, this resource facilitates the rapid identification of candidate lipid-binding proteins and supports orthogonal validation efforts.</p>
<p>Designed as a continuously expanding repository, the Lipid Interactome will evolve alongside advances in lipid probe technologies and interactomics. Ongoing efforts seek to incorporate enhanced scalability of this site, more robust data storage methods, and automated data submission. These functionalities will improve the utility of this repository, enabling applications such as machine learning for pattern recognition as the site grows. Future iterations will incorporate validation studies for each lipid probe, offering researchers a dedicated space to compare findings across experimental contexts. This functionality will not only reduce redundancy in experimental design but also strengthen the reproducibility and biological interpretation of lipid interactome research.</p>
<p>To support this vision, we invite researchers to contribute their datasets via our <a href="https://lipidinteractome.org/contactus/datasubmission">Data Submission page</a>. Using the FAIR data principles, we hope to foster a collaborative and transparent approach to lipid biology.</p>
</section>
<section id="abbreviations" class="level2">
<h2 class="anchored" data-anchor-id="abbreviations">Abbreviations</h2>
<dl>
<dt>UV</dt>
<dd>
<em>U</em>ltra<em>V</em>iolet
</dd>
<dt>MS</dt>
<dd>
<em>M</em>ass <em>S</em>pectrometry
</dd>
<dt>LIMMA</dt>
<dd>
<em>Li</em>near <em>M</em>odels for <em>M</em>icro<em>a</em>rray and RNA-Seq Data
</dd>
<dt>ANOVA</dt>
<dd>
<em>An</em>alysis <em>o</em>f <em>Va</em>riance
</dd>
<dt>PSM</dt>
<dd>
<em>P</em>eptide <em>S</em>pectral <em>M</em>atching
</dd>
<dt>DIA</dt>
<dd>
<em>D</em>ata-<em>I</em>ndedpendent <em>A</em>cquisition
</dd>
<dt>DDA</dt>
<dd>
<em>D</em>ata-<em>D</em>edpendent <em>A</em>cquisition
</dd>
</dl>
</section>
<section id="contributions" class="level2">
<h2 class="anchored" data-anchor-id="contributions">Contributions</h2>
<p>GG designed, implemented, and wrote the majority of the Lipid Interactome repository and manuscript. FS wrote the data analysis section of the repository, provided significant intellectual contribution to the repository design, and edited the manuscript. AN provided substantial editing of the repository and manuscript. JMB edited the manuscript and contributed data to the repository. CS and FT provided significant intellectual contribution and text editing of the repository and manuscript and oversaw the site development.</p>
</section>
<section id="funding" class="level2">
<h2 class="anchored" data-anchor-id="funding">Funding</h2>
<p>This work was supported by funding from the National Institutes of Health (NIAID R01 AI141549-02, to FGT, CS; NIGMS R01 GM127631 to C.S., and NIGMS R01 GM151682 to JMB).</p>
</section>
<section id="conflicts-of-interest" class="level2">
<h2 class="anchored" data-anchor-id="conflicts-of-interest">Conflicts of Interest</h2>
<p>The authors have no conflicts of interest to declare.</p>
<p><br></p>
</section>
<section id="citations" class="level2">
<h2 class="anchored" data-anchor-id="citations">Citations</h2>
<div id="refs" role="list">

</div>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>