---
title: "Data Analysis and Statistics"
---

## TMT quantitative mass spec pipeline

To ensure consistency across all publicly available mass spectrometry datasets that used TMT isobaric mass tagging, we applied the same standardized analysis pipeline previously published. This approach allows for better comparability between datasets. Several of the datasets included in this repository were generated and analyzed at the European Molecular Biosciences Laboratories (EMBL) in Heidelberg, Germany, where Dr. Frank Stein developed and implemented the data analysis pipeline.
In summary, we processed the mass spectrometry (MS) data using FragPipe[@yuAnalysisDIAProteomics2023], following established methodologies outlined in Farley et al. (2024, A), Farley et al. (2024, B), and Thomas et al. (2025)[@farleyTrifunctionalSphinganineNew2024a; @farleyTrifunctionalFattyAcid2024; @thomasTrifunctionalLipidDerivatives2025]. To ensure accurate protein identification, we used a UniProt FASTA database that included common contaminants and reversed sequences for validation. Proteins were included in the final dataset only if they contained at least two unique peptides, each at least seven amino acids long.

To account for potential technical variations across experiments, we applied a normalization step using the 'removeBatchEffect' function from the limma package, stabilizing variance in log2-transformed TMT reporter ion intensities[@huberVarianceStabilizationApplied2002; @ritchieLimmaPowersDifferential2015]. Further normalization was carried out using the 'normalizeVSN' function of the same package. We identified differentially expressed proteins using the moderated t-test within limma, incorporating replicate information into the statistical model through the 'lmFit' function.

For the protein-lipid probe interaction experiments (+/- UV exposure), proteins were categorized based on their enrichment in the +UV condition. Proteins showing a log2 fold-change of at least 1 and a p-value below 0.05 were classified as "enriched hits," while those showing similar enrichment trends but with p-values above 0.05 were labeled as “enriched candidates.”

To better understand the biological significance of the detected proteins, we conducted gene ontology (GO) enrichment analysis using the 'compareCluster' function from the 'clusterProfiler' package[@yuClusterProfilerPackageComparing2012]. This method evaluates whether specific GO terms are overrepresented in the dataset compared to a background gene set. We performed enrichment analysis for three GO categories: Cellular Component (CC), Molecular Function (MF), and Biological Process (BP). The reference database used was 'org.Pf.plasmo.db.' To quantify enrichment, we calculated the odds ratio by comparing the proportion of genes associated with each GO term in our dataset ('GeneRatio') with the proportion in the reference background set ('BgRatio'). A value greater than 1 suggests that a given GO term is more prevalent in our dataset than expected by chance.


## PSM quantitative mass spec analysis

* Studies included in this repository which used a PSM-based analysis pipeline
	- Hoglinger et al., 2017[@hoglingerTrifunctionalLipidProbes2017]
	- Muller et al., 2021[@mullerSynthesisCellularLabeling2021]

* Need these details.